scripts/setup/ubuntu/setup.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+# Bootstrap script for Ubuntu
+
+source ./scripts/lib/echoes.sh
+source ./scripts/lib/utils.sh
+
+install_packages() {
+
+  bot "The following packages are going to be installed\n"
+
+  cat ./scripts/setup/ubuntu/packages.txt
+
+  echo
+
+  read -p "Do you want to proceed? [y/N] " -n 1 answer
+  echo
+  if [[ ! $answer =~ (yes|y|Y) ]];then
+    exit 1
+  fi
+
+  xargs -a ./scripts/setup/ubuntu/packages.txt sudo apt install -y
+
+  ok "Installation finished!"
+}
+
+install_1password() {
+  bot "Checking 1password..."
+  echo
+  if ! _exists 1password ; then
+    read -p "Would you like to install 1password? [y/N]" -n 1 answer
+    echo
+    if [[ $answer =~ (yes|y|Y) ]] ;then
+      action "Adding key for apt repo..."
+      curl -sS https://downloads.1password.com/linux/keys/1password.asc | sudo gpg --dearmor --output /usr/share/keyrings/1password-archive-keyring.gpg
+      action "Adding apt repo..."
+      echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/1password-archive-keyring.gpg] https://downloads.1password.com/linux/debian/amd64 stable main' | sudo tee /etc/apt/sources.list.d/1password.list
+      action "Adding the debsig-verify policy..."
+      sudo mkdir -p /etc/debsig/policies/AC2D62742012EA22/
+      curl -sS https://downloads.1password.com/linux/debian/debsig/1password.pol | sudo tee /etc/debsig/policies/AC2D62742012EA22/1password.pol
+      sudo mkdir -p /usr/share/debsig/keyrings/AC2D62742012EA22
+      curl -sS https://downloads.1password.com/linux/keys/1password.asc | sudo gpg --dearmor --output /usr/share/debsig/keyrings/AC2D62742012EA22/debsig.gpg
+      action "Installing 1password..."
+      sudo apt update && sudo apt install -y 1password
```

Comment:
@lfrezarini Why is ubuntu so complicateeeed?? On arch I can just run `yay -S 1password` and avoid all this shit.